content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,136 @@
+---
+title: AsyncAPI Code Generation: Microservices Using Spring Cloud Stream
+published: true
+date: 2020-04-28 17:39:34 UTC
+tags: solace, asyncapi, java, spring, EDA
+authors:
+   - name: Marc DiPasquale
+     photo: /images/avatars/marc-dipasquale.webp
+     link: https://twitter.com/mrc0113
+     byline: Developer Advocate at Solace
+---
+
+ ![](https://solace.com/wp-content/uploads/2020/04/blog-featured-image-asyncapi.jpg) 
```

Comment:
embedding remove images is super risky and we also prefer `.webp`
and anyway, is this picture really needed here?

Reply:
removing image